pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +77,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from a binary file via the DC/OS Enterprise CLI
+
+This procedure describes how to use any file to create a secret called `myapp.keystore` inside the `developer` path using the DC/OS Enterprise CLI.
```

Comment:
This says "any file" but the title says "a binary file" which [Wikipedia tells me](https://en.wikipedia.org/wiki/Binary_file) is "a computer file that is not a text file".

Reply:
Changed to:

> The contents of the file (referred to below as `myapp.keystore`) can be any value, including binary and text value.